api/src/test/resources/org/openmrs/module/idgen/include/TestData.xml

```diff
@@ -7,8 +7,11 @@
     <IDGEN_IDENTIFIER_SOURCE ID="4" UUID="0d42284f-9e9b-4a81-a88b-8bb42bc0a904" NAME="Another Test Identifier Pool" IDENTIFIER_TYPE="4" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
 	<IDGEN_IDENTIFIER_SOURCE ID="5" UUID="0d42284f-9e9b-4a81-a88b-8bb42bc0a905" NAME="Id Pool Filled from Remote" IDENTIFIER_TYPE="4" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
 	<IDGEN_IDENTIFIER_SOURCE ID="6" UUID="0d42284f-9e9b-4a81-a88b-8bb42bc0a906" NAME="Remote to fill Id Pool" IDENTIFIER_TYPE="4" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+	<IDGEN_IDENTIFIER_SOURCE ID="7" UUID="0d47284f-9e9b-4a81-a88b-8bb42bc0a907" NAME="Local Sequential Generator" IDENTIFIER_TYPE="2" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
 
-	<IDGEN_SEQ_ID_GEN ID="1" first_identifier_base="1" base_character_set="ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789" PREFIX="" min_length="0" max_length="0"/>

```

Comment:
If we choose not to expose this per comments above, we should just set up test data in code